src/transport_picoquic.h

```diff
@@ -132,14 +133,17 @@ class PicoQuicTransport : public ITransport
                       uint8_t* bytes, size_t length);
 
 
+    void update_datagram_ready_status(picoquic_cnx_t *cnx, bool value);
+
     /*
    * Internal Public Variables
      */
     LogHandler& logger;
     bool _is_server_mode;
     bool _is_unidirectional{ false };
     bool debug {false};
-
+    // current status of our knowledge of picoquic datagram ready status set
+    std::atomic<bool> pq_datagram_ready { false };
```

Comment:
it is prefixed with pq since it is picoquic api status .. also we at some point  in Jan agreed not to use _XXX and mXXX form of naming .. again i don't care either way

Reply:
> not to use _XXX and mXXX form of naming .. again i don't care either way

:(